README.md

```diff
@@ -93,54 +94,54 @@ After doing the above steps, navigate to `http://localhost:1331` to see the Serv
 
 #### Vue.js
 
- - `dev` - runs `vite` that starts development server doing hot reload over source files
+ - `dev` - runs `vite` that starts the development server doing hot reload over source files
  - `build` - runs build script that outputs files to `..\app` folder
- - `lint` - checks with eslint all vue, ts and js files
+ - `lint` - checks with eslint all Vue, ts and js files
  - `type-check` - runs TypeScript in no emit mode
 
 #### AngularJS
 
  - `test` - runs js tests in ServicePulse.Host.Test project
- - `setup` - this command runs few commands
+ - `setup` - this command runs a few commands
  	- installs the npm packages
  	- runs `webpack`  
- - `load` - this command does the same thing that `setup` but webpack is run only once to produce artifacts. This command is used by the builder
+ - `load` - this command does the same thing as `setup`, but webpack is run only once to produce artifacts. The builder uses this command
  - `serve` - serves js artifacts using `http-server`
  - `webpack` - runs webpack command, which does the following things for the given modules (monitoring, configuration)
 	 - joins js together 
 	 - runs babel transpiler 
-	 - has a file watcher to run the above whenever file is saved
+	 - has a file watcher to run the above whenever a file is saved
  - `lint` - checks with eslint all js files
- - `dev` - runs in parallel two scripts `webpack` and `serve`
+ - `dev` - runs in parallel two scripts, `webpack` and `serve`
 
 NOTE:
-Webpack observes files and updates the artifacts whenever they are changed, however at the moment not every bit of code is processed by webpack. Only monitoring and configuration is.
+Webpack observes files and updates the artifacts whenever they are changed, however at the moment not every bit of code is processed by webpack. Only monitoring and configuration-related files are.
 
 ### Configuring automated tests
 
-For information how to run automated tests please follow [ServicePulse.Host.Tests/Readme](https://github.com/Particular/ServicePulse/blob/master/src/ServicePulse.Host.Tests/README.md).
+For information on how to run automated tests, please follow [ServicePulse.Host.Tests/Readme](https://github.com/Particular/ServicePulse/blob/master/src/ServicePulse.Host.Tests/README.md).
 
 ## Running from ServicePulse.Host.exe
 
 It is possible to run ServicePulse directly via `ServicePulse.Host.exe`.
 
 ### Step 1 - reserve URL ACL
 
-ServicePulse.Host.exe depends on a self-hosted webserver. In order to start the project a URL ACL reservation needs to be setup. Either run Visual Studio with Administrative privileges or run the following command to add the required URL ACL reservation:
+ServicePulse.Host.exe depends on a self-hosted web server. A URL ACL reservation needs to be set up to start the project. Either run Visual Studio with Administrative privileges or run the following command to add the required URL ACL reservation:
```

Comment:
@cquirosj, do you happen to know the URL ACL reservation command line command? It's missing from the README

Reply:
https://learn.microsoft.com/en-us/windows-server/networking/technologies/netsh/netsh-http#add-urlacl